NCEP/run_graphcast.py

```diff
@@ -137,14 +146,40 @@ def get_predictions(self, fname, forecast_length):
 
         print (f"start running GraphCast for {forecast_length} steps --> {forecast_length*6} hours.")
         self.load_model()
-            
+           
         # output = self.model(self.model ,rng=jax.random.PRNGKey(0), inputs=self.inputs, targets_template=self.targets * np.nan, forcings=self.forcings,)
-        forecasts = rollout.chunked_prediction(self.model ,rng=jax.random.PRNGKey(0), inputs=self.inputs, targets_template=self.targets * np.nan, forcings=self.forcings,)
+        forecasts = rollout.chunked_prediction(self.model, rng=jax.random.PRNGKey(0), inputs=self.inputs, targets_template=self.targets * np.nan, forcings=self.forcings,)
         
+        # check output format
+        if fname.endswith('grib2'):
+
+            path = pathlib.Path(fname)
+            outdir = path.parent
+
+            # drop "batch" from variables
+            for var in forecast.variables:
+                if 'batch' in forecast[var].dims:
+                    forecast[var] = forecast[var].squeeze(dim='batch')
+
+            #update "level" attributes
+            forecast['level'] = forecast['level'] * 100
+            forecast['level'].attrs['long_name'] = 'pressure'
+            forecast['level'].attrs['units'] = 'Pa'
+
+            #save netcdf to file
+            new_fname = f"{fname.split('.')[0]}.nc"
```

Comment:
@LinlinCui-NOAA Please do not rename netcdf forecasts

Reply:
I'll do this after the original netcdf file being saved and save it as an intermediate file. Otherwise, Iris cannot recognize dim "batch". And the unit of "level" must be "Pa" instead of "mb".